packages/@okta/vuepress-site/docs/guides/submit-app/submission2-specific/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Configure protocol specific settings
+---
+
+Your app has to support at least one protocol: SAML or OIDC for authentication, or SCIM for provisioning.
+
+If you want to support more than one, you can submit asynchronously. For example, you can submit your SAML app now and then add OIDC configuration details later on when your app is configured for OIDC.
```

Comment:
"asynchronously" confuses me. Do you mean submit the app integration now for just one of the protocols, then update the integration later?

Reply:
Yes, that's what was intended - you can submit your SAML-enabled version now, then come back at some later time and add the OIDC specific instructions when your app supports it.

Reply:
Marking this resolved with the paragraph changes from the comment below.